cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:$TAG_NAME', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:latest', '.']
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
I'm not very familiar with Cloud Build, but do you actually need this step?

https://cloud.google.com/cloud-build/docs/build-config#images

Looking at the documentation above, it seems like the images listed in `images` are pushed by default.

Reply:
>  if you build images and don't push them to Container Registry, the images are discarded on build completion

I saw the push thing [here](https://cloud.google.com/cloud-build/docs/building/build-containers#storing_images_in). 🤷‍♂️

Also not too familiar but we need `latest` tag for the CLI to not hardcode a tag.

Reply:
>The difference between using the `images` field and the Docker `push` command is that if you use the images field, the stored image will be displayed in the build results.

Looks like you specifying only the `images` field should be enough.

Reply:
Yeah, I think you are right. "as part of your build flow" part confused me. I thought that without the pushing the images will be available only within a single run in case you multiple things using the image down the "pipeline".